educator/src/Dscp/DB/SQLite/Util.hs

```diff
@@ -226,15 +239,14 @@ filterMatchesPk_ = \case
 -- to do the similar thing.
 getNextPrimaryKey
     :: (PrimaryKeyWrapper (PrimaryKey table Identity) keyInner
-       ,MonadIO m
        ,Num keyInner
+       ,Beam.MonadBeam cmd be hdl m
```

Comment:
That ' s not now semicolons are used ,I think .

Reply:
Mm yes, my hindent is definitely mad here.

Reply:
Yeah, them